client/src/app/shared/models/agenda/item.ts

```diff
@@ -72,4 +72,12 @@ export class Item extends ProjectableBaseModel {
     public getProjectorTitle(): string {
         return this.getListTitle();
     }
+
+    public getVisibility(): string {
```

Comment:
Mavbe make a getter `verboseType`, to keep the wording equal. Also this should return `''` instead of `null`.